### PR TITLE
Minor knife changes.

### PR DESCRIPTION
**Pull request by harmonyn**
- Carrot shivs no longer contain metal or origin tech. Added flavor verbs.
- Cleavers have 10 throwforce, a minor increase from 8.
- Knives can now be used to stab people in the eyes like screwdrivers.

### CODE REVIEW

**Comment by HankMoodyExperience** [comment]
nice

**Comment by bgobandit** [comment]
would anyone be opposed to my generalizing the eye stabbing to all IS_SHARP objects

File: code/game/objects/items/weapons/kitchen.dm [hunk 1]
```diff
@@ -65,6 +65,14 @@
 	attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
 	sharpness = IS_SHARP_ACCURATE
 
+/obj/item/weapon/kitchen/fork/attack(mob/living/carbon/M, mob/living/carbon/user)
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/kitchen.dm]: dat copy paste =p

**Comment by RandomMarine** [comment]
That would mean eyestabs with shovels and butcher cleavers @bgobandit

**Comment by TheNightingale** [comment]
I think if I got stabbed in the eye with a shovel, it'd hurt too.

**Comment by Supermichael777** [comment]
eyestab needs to be on narrow objects only

**Comment by PKPenguin321** [comment]
>Carrot shivs no longer contain metal or origin tech

Haha whoops

File: code/game/objects/items/weapons/kitchen.dm [hunk 2]
```diff
@@ -65,6 +65,14 @@
 	attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
 	sharpness = IS_SHARP_ACCURATE
 
+/obj/item/weapon/kitchen/knife/attack(mob/living/carbon/M, mob/living/carbon/user)
+	if(user.zone_selected == "eyes")
```
Comment (KorPhaeron) [inline code/game/objects/items/weapons/kitchen.dm]: This means it bypasses shield/armour if you aim for the eyes with a knife, doesn't it?

Comment (bgobandit) [inline code/game/objects/items/weapons/kitchen.dm]: no, eyestab handles that

**Pull request merged**